src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -880,38 +916,38 @@ class ProjectReview extends Component {
           button({
             className: "btn buttonPrimary floatRight",
             onClick: this.submitEdit(),
-            disabled: this.compareObj("formData", "current") || this.compareObj("formData","editedForm"),
+            disabled: this.compareObj("formData", "current") || this.compareObj("formData", "editedForm"),
             isRendered: this.state.readOnly === false
           }, ["Submit Edits"]),
 
+          /*visible for Admin in readOnly mode and if the project is in "pending" status*/
+          button({
+            className: "btn buttonPrimary floatRight",
+            onClick: this.handleApproveInfoDialog,
+            disabled: this.state.disableApproveButton,
+            isRendered: this.isAdmin() && projectReviewApproved === 'false' && this.state.readOnly === true
+          }, ["Approve"]),
+
           /*visible for Admin in readOnly mode and if there are changes to review*/
           button({
             className: "btn buttonPrimary floatRight",
             onClick: this.handleApproveDialog,
-            isRendered: this.isAdmin() && this.state.reviewSuggestion
+            isRendered: this.isAdmin() && this.state.reviewSuggestion && (this.state.readOnly === true && projectReviewApproved === 'true')
           }, ["Approve Edits"]),
 
-          /*visible for every user in readOnly mode and if there are changes to review*/
-          button({
-            className: "btn buttonSecondary floatRight",
-            onClick: this.handleDiscardEditsDialog,
-            isRendered: this.isAdmin() && this.state.reviewSuggestion
-          }, ["Discard Edits"]),
-
           /*visible for Admin in readOnly mode and if the project is in "pending" status*/
           button({
-            className: "btn buttonPrimary floatRight",
-            onClick: this.approveRevision(),
-            disabled: this.state.disableApproveButton,
-            isRendered: this.isAdmin() && !this.state.formData.projectExtraProps.projectReviewApproved
-          }, ["Approve"]),
+            className: "btn buttonSecondary floatRight",
+            onClick: this.handleRejectProjectDialog,
+            isRendered: this.isAdmin() && projectReviewApproved === 'false' && this.state.readOnly === true
```

Comment:
Similar feedback to above, this would read a lot better as 
```suggestion
            isRendered: this.isAdmin() && !projectReviewApproved && this.state.readOnly
```